dewy/collection/router.py

```diff
@@ -83,3 +83,22 @@ async def get_collection(name: PathCollection, conn: PgConnectionDep) -> Collect
         )
 
     return Collection.model_validate(dict(result))
+
+@router.delete("/{name}")
+async def delete_collection(pg_pool: PgPoolDep, name: PathCollection) -> Collection:
+    """Delete a collection and all documents contained within it."""
+    async with pg_pool.acquire() as conn:
```

Comment:
FWIW, you don't need to acquire a connection in cases like this. You can run directly on `pg_pool` (it will automatically acquire a connection and run on that).

You could also pass the `pg_conn: PgConnDep` directly in, and not fool with the pool.